### ISSUE TITLE
Mangas Chan Blocked by cloudflare

### ISSUE THREAD

**Issue by Doug-Soares** [keiyoushi/tachiyomi-extensions-source#289]
### Source information

Mangas Chan 1.4.28

### Source language

Portuguese (Brazil)

### Steps to reproduce

The extension isn't working because the app can't pass the cloudflare

### Expected behavior

Show mangas and can open the site in webview

### Actual behavior

Don't show any manga and when you open in webview show the message "sorry, you have been blocked"

### Tachiyomi version

0.15.3

### Android version

Android 11

### Other details

I already tried to change the user agent and all the DNS options in the app, clear cookies, clear webview data and this problem still persisting 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
